frontend/routes/Wifi.jsx

```diff
@@ -73,14 +70,9 @@ const headCells = {
   SECURITY: { numeric: false, disablePadding: false, label: 'Security' },
 };
 
-// Meta-Data 
-
-
-function EnhancedTableHead(props) {
+function SortableTableHead(props) {
   const { classes, order, orderBy, onRequestSort } = props;
-  const createSortHandler = property => event => {
-    onRequestSort(event, property);
-  };
+  const createSortHandler = property => event => onRequestSort(event, property);
```

Comment:
By inline I meant instead of creating a function here with just a single use, just write this directly in the `onClick` handler, sorry if that was unclear!

Reply:
Done